src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java

```diff
@@ -404,7 +407,7 @@ synchronized public void changeClaimOwner(Claim claim, UUID newOwnerID)
 
         //determine current claim owner
         PlayerData ownerData = null;
-        if (!claim.isAdminClaim())
+        if (!claim.isAdminClaim() && claim.ownerID != null)
```

Comment:
Theoretically it's possible for a non-admin claim to have a null owner ID if an addon creates a subclaim with a null owner ID.